cache/clock_cache.cc

```diff
@@ -618,14 +620,24 @@ Cache::Handle* ClockCacheShard::Lookup(const Slice& key, uint32_t hash) {
   return reinterpret_cast<Cache::Handle*>(handle);
 }
 
-void ClockCacheShard::Release(Cache::Handle* h) {
+bool ClockCacheShard::Release(Cache::Handle* h, bool force_erase) {
   CleanupContext context;
   CacheHandle* handle = reinterpret_cast<CacheHandle*>(h);
   Unref(handle, true, &context);
   Cleanup(context);
+  auto erased = context.to_delete_value.size() > 0;
+  if (!force_erase || erased) {
+    return erased;
+  } else {
+    return EraseAndConfirm(handle->key, handle->hash);
```

Comment:
@yiwu-arbug I think this implementation is correct. Feel free to suggest a more optimized version if you see it applies. Thanks